### PR TITLE
v4.1.0

### PR DESCRIPTION
**Pull request by trygve-lie**
Update doc to reflect the v4.1.0 changes. Also added a blog post about the release.

### CODE REVIEW

File: docs/api/podlet.md [hunk 1]
```diff
@@ -1170,16 +1170,17 @@ podlet.fallback({ prefix: true });
...
-Sets a relative or absolute URL to a JavaScript asset for the podlet.
+Set relative or absolute URLs to JavaScript asset for the podlet.
 
-When a value is set it will be internally kept and used when the podlet
-instance is serialized into a manifest JSON string. The value will also be
-available for the document template to include. The method can be called
-multiple times to set multiple values.
+When set the values will be internally kept and made available for the document
+template to include. The assets set are also made available in the manifest for
+the Layout to consume.
```
Comment (digitalsadhu) [inline docs/api/podlet.md]: layout

File: website/blog/2019-07-02-version-4.1.0.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+author: Trygve Lie
+authorURL: https://github.com/trygve-lie/
+---
+
+Version 4.1.0 of `@podium/layout` and `@podium/podlet` are now available.
+
+### Assets
+
+This release contain some minor changes to the `.js()` and `.css()` methods in
+both `@podium/layout` and `@podium/podlet` which paves ground for work we are
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: paves the way for the work we are doing

File: website/blog/2019-07-02-version-4.1.0.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+doing to improve asset handling and bundling when building microfrontends with
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: is it micro-frontends, micro frontends or microfrontends?

File: website/blog/2019-07-02-version-4.1.0.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Podium.
+
+The changes are:
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: These changes

File: website/blog/2019-07-02-version-4.1.0.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Currently `.js()` and `.css()` returns its target value when called. This is
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: Currently the `.js()` and `.css()` methods return their target values...

File: website/blog/2019-07-02-version-4.1.0.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+now deprecated and the methods will in close future not have a return value.
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: these methods will cease to return a value in the near future

File: website/blog/2019-07-02-version-4.1.0.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+If you are doing stuff like this:
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: doing something like this

File: website/blog/2019-07-02-version-4.1.0.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+```js
+app.get(podlet.js({ value: '/assets.js' }), (req, res) => {
+    res.status(200).sendFile('./src/js/main.js', err => {});
+});
+```
+
+You should rewrite it to do as follow:
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: ...rewrite it to the following

File: website/blog/2019-07-02-version-4.1.0.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    { value: '/assets/extra.js' },
+]);
+```
+
+We will write more about our work on asset handling and bundling when we have
+some more concrete code to show.
+
+### Proxy
+
+This release does also contain a small fix to the proxy preventing it from
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: release also contains a...

File: website/blog/2019-07-02-version-4.1.0.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+resolving a proxy request as successfull after a failed proxy request occured.
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: successful ... has occurred

File: website/blog/2019-07-02-version-4.1.0.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+This mostly affected metrics causing failed proxy requests to also be counted
+as successfull requests.
```
Comment (digitalsadhu) [inline website/blog/2019-07-02-version-4.1.0.md]: successful

File: website/blog/2019-07-02-version-4.1.0.md (hunk 3 above)
Comment (trygve-lie) [reply website/blog/2019-07-02-version-4.1.0.md]: I guess its microfrontends

**Pull request merged**